assets/templates/partials/census/variables-table.tmpl

```diff
@@ -26,39 +26,53 @@
                         </th>
                         <td class="ons-table__cell ons-u-pb-s ons-u-pt-s ons-u-pl-no@xxs@s ons-u-order--3 ons-u-fw@xxs@s ons-u-pt-no@xxs@s
                                 ons-u-bb-no@xxs@s">
-                            {{ if or ($dim.IsAreaType) ($dim.IsCoverage) }}
+                            {{ if or ($dim.IsAreaType) ($dim.IsDefaultCoverage) }}
                                 <div>
-                                    {{ if $dim.IsCoverage }}
+                                    {{ if $dim.IsDefaultCoverage }}
                                         {{- localise "AreaTypeDefaultCoverage" $language 1 -}}
                                     {{ else }}
                                         {{- $dim.Title }} ({{ $dim.TotalItems -}})
                                     {{ end }}
                                 </div>
                             {{else}}
-                                {{ $strOptCount := intToString $dim.TotalItems }}
-                                <div>
-                                    {{ $dim.TotalItems }}
-                                    {{ if gt $dim.TotalItems 1 }}
-                                        {{ localise "Category" $language 4 }}
-                                    {{else}}
-                                        {{ localise "Category" $language 1 }}
-                                    {{end}}
-                                </div>
-                                <div class="ons-u-mt-s ons-u-fs-s ons-list--container">
-                                    <ul class="ons-list{{ if $dim.IsTruncated }}--truncated{{end}}{{ if or (gt $dim.TotalItems 9) ($dim.IsTruncated) }}
-                                            ons-u-mb-xs{{else}} ons-u-m-no{{end}}">
-                                        {{ range $j, $val := $dim.Values}}
-                                            <li class="ons-list__item{{ if $dim.IsTruncated }}--truncated{{end}} ons-u-mb-no">
-                                                {{- $val -}}
-                                            </li>
+                                {{ if .IsCoverage }}
+                                    {{ if $dim.Values }}
```

Comment:
What happens if there is coverage but no $dim.Values?

Reply:
The `IsDefaultCoverage` should be `true` but the guard clause is primarily there to make sure the service doesn't panic on line 47 ` {{- index $dim.Values 0 -}}`